src/utils/myHeaders.js

```diff
@@ -0,0 +1,6 @@
+const token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6IndlbGlrZS5hbW9zQGFuZGVsYS5jb20iLCJleHAiOjE1NDE3NTQ5MDMsInVzZXJuYW1lIjoiamFrZSJ9.HCXIuZz8Ty8W5Qlt57qGe7_egeVv2YZJfMoEuKGYRw4";
+export var myHeaders = new Headers({
```

Comment:
Replace the token with an environment variable as well

Reply:
Token has been replaced with an environment variable